content/docs/setup/kubernetes/helm-install.md

```diff
@@ -111,6 +112,38 @@ The Helm chart also offers significant customization options per individual
 service. Customize these per-service options at your own risk. The per-service options are exposed via
 the [`values.yaml`](https://raw.githubusercontent.com/istio/istio/{{<branch_name>}}/install/kubernetes/helm/istio/values.yaml) file.
 
+## Customization example: traffic management minimal set
+
+Istio is equipped with a rich and powerful set of features and some users may need only subset of those. For instance, users might be interested only
+in installing the minimal set required to Istio's traffic management.
+[Helm customization](#customization-with-helm) provides the option to install a subset by enabling those features of interest and disabling the ones that aren't required.
+
+In this example we will install Istio with only a minimal set of components necessary to conduct [traffic management](/docs/tasks/traffic-management/).
+
+Execute the following command to install the Pilot, Citadel, IngressGateway and Sidecar-Injector:
+
+```command
+$ helm install install/kubernetes/helm/istio --name istio --namespace istio-system \
+  --set ingress.enabled=false,egressgateway.enabled=false,galley.enabled=false \
+  --set mixer.enabled=false,prometheus.enabled=false,global.proxy.envoyStatsd.enabled=false
+```
+
+Ensure the following Kubernetes pods are deployed and their containers are up and running: `istio-pilot-*`, `istio-ingressgateway-*`,
+`istio-citadel-*` and `istio-sidecar-injector-*`.
+
+```command
+$ kubectl get pods -n istio-system
+NAME                                     READY     STATUS    RESTARTS   AGE
+istio-citadel-b48446f79-wd4tk            1/1       Running   0          1m
+istio-ingressgateway-7b77d995f7-t6ssx    1/1       Running   0          1m
+istio-pilot-58c65f74bc-2f5xn             2/2       Running   0          1m
+istio-sidecar-injector-86cc99578-4t58m   1/1       Running   0          1m
+```
+
+With this minimal set you can proceed to installing the sample [Bookinfo](/docs/guides/bookinfo/) application or install your own application and [configure request routing](/docs/tasks/traffic-management/request-routing/) for instance.
+
+Of course that if no ingress is expected and sidecar is to be [injected manually](/docs/setup/kubernetes/sidecar-injection/#manual-sidecar-injection) then you can reduce this minimal set even further and only have Pilot and Citadel. However, Pilot depends on Citadel therefore you can't install it without the other.
+
```

Comment:
This looks pretty good.  I'd also suggest adding a section about operating with nodeport, as the documentation is currently very nodeport heavy.

I use this to launch Istio:
```
helm install upstream/istio --name istio --namespace istio-system --set ingress.enabled=false --set ingressgateway.service.type=NodePort
```

To add nodeport, just need `ingressgateway.service.type=NodePort` in a sec argument.

Cheers
-steve